drv/transceivers-server/src/udp.rs

```diff
@@ -334,13 +335,14 @@ impl ServerImpl {
                 ringbuf_entry!(Trace::OperationResult(result));
                 let success = ModuleId::from(result.success());
                 let read_bytes = result.success().count() * read.len() as usize;
-                let (final_payload_len, failed_modules) = self
+                let (err_len, failed_modules) = self
                     .handle_errors_and_failures(
                         modules,
                         result,
                         HwError::I2cError,
                         &mut out[read_bytes..],
                     );
+                let final_payload_len = num_status_bytes + err_len;
```

Comment:
Okay, now this one is wrong – I think it should be `read_bytes + err_len`

Reply:
notice that locally and fixed it in a follow up